javascript/main.js

```diff
@@ -9,52 +10,105 @@ var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
 
 
-function printTime() {
+function setCallback(){
+  chronometer.setPrintTime(printTime);
+};
+
+function printTime(time) {
+  printMinutes(time[0]);
+  printSeconds(time[1]);
 
 }
 
-function printMinutes() {
+function printMinutes(minutes) {
+  minDec.innerHTML = minutes[0];
+  minUni.innerHTML = minutes[1];
 
 }
 
-function printSeconds() {
-
+function printSeconds(seconds) {
+  secDec.innerHTML = seconds[0];
+  secUni.innerHTML = seconds[1];
 }
 
 function printMilliseconds() {
 
 }
 
 function printSplit() {
-
+  let list = document.createElement("li");
+  list.innerHTML = chronometer.setTime();
+  list.setAttribute("class", "listItems");
+  olist.appendChild(list);
 }
 
 function clearSplits() {
-
+  var elem1 = document.querySelectorAll('.listItems');
+  var elem = [...elem1];
+  elem.forEach(function(element){element.parentNode.removeChild(element);
+});
 }
 
 function setStopBtn() {
-
+  btnLeft.innerHTML ="STOP";
+  btnLeft.classList.toggle("btnstop");
+  btnLeft.style.backgroundColor = "red";
```

Comment:
the way the css and html were set up the buttons had two classes, one being "btn" and the other "start" or "stop", so the css handled the the different colors and everything.